scanner/src/main/kotlin/CloudStorageCache.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.scanner
+
+import ch.frankel.slf4k.*
+import com.google.auth.oauth2.ServiceAccountCredentials
+import com.google.cloud.storage.Storage
+import com.google.cloud.storage.StorageOptions
+import com.here.ort.model.*
+
+import com.here.ort.model.config.CloudStorageCacheConfiguration
+import com.here.ort.utils.log
+import java.io.FileInputStream
+
+class CloudStorageCache(
```

Comment:
I think this class should be called `GoogleCloudStorageCache`, because "cloud storage" is a very generic term.

Reply:
Yeah I was super torn on this one, on one hand usually you would not say Amazon S3, it's just S3, same in ORT code with Artifactory, it's just Artifactory and not JFrog Artifactory, but you're right it's quite generic. 

@sschuberth Thoughts?

Reply:
I agree it should be called `GoogleCloudStorageCache`. "Cloud" is just a generic English word, whereas "Artifactory" is not (it's a made up product name).

Reply:
@sschuberth @mnonnenmacher renamed